pkg/scheduler/framework/plugins/imagelocality/image_locality.go

```diff
@@ -97,7 +160,8 @@ func calculatePriority(sumScores int64, numContainers int) int64 {
 func sumImageScores(nodeInfo *framework.NodeInfo, containers []v1.Container, totalNumNodes int) int64 {
 	var sum int64
 	for _, container := range containers {
-		if state, ok := nodeInfo.ImageStates[normalizedImageName(container.Image)]; ok {
+		state, ok := nodeInfo.ImageStates[normalizedImageName(container.Image)]
+		if ok && container.ImagePullPolicy != v1.PullAlways {
```

Comment:
This is impossible, right? For we'll early return in Score() if we have containers with `imagePullPolicy=Always`.

Reply:
see: https://github.com/kubernetes/kubernetes/pull/110382#discussion_r1096052515